target/linux/mpc85xx/files/arch/powerpc/boot/dts/ecw7210.dts

```diff
@@ -0,0 +1,304 @@
+/*
+ * Edge-core ECW-7210-L Device Tree Source
+ *
+ * Copyright (C) 2019 Scott Gilliland <zeroping@gmail.com>
+ *
+ * This program is free software; you can redistribute  it and/or modify it
+ * under  the terms of  the GNU General  Public License as published by the
+ * Free Software Foundation;  either version 2 of the  License, or (at your
+ * option) any later version.
+ */
+
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
You arleady have a license above. Remove the above version and stick to the license identifier here.